packages/script/install.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+chmod 775 -R /opt/yugabyte-2.0.5.2
+cd /opt/yugabyte-2.0.5.2/
+
+mkdir -p /var/log/yugabyte-logs/
+mv /opt/yugabyte-2.0.5.2/etc/yugabyte /etc
+
+rm -rf /opt/yugabyte-2.0.5.2/etc
+ln -s /opt/yugabyte-2.0.5.2/yugabyted /usr/bin/yugabyted
```

Comment:
as part of installation, could we run `/opt/yugabyte-<version>/bin/post_install.sh && touch /opt/yugabyte-<version>/post_install.sh.completed`

Reply:
added this in `install.sh` script.